src/components/ProductCard.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from "@emotion/styled";
+
+interface ProductCardProps {
+    item: {
+        id: number | string;
+        name: string;
+        imageURL: string;
+        price: {
+            sellingPrice: number;
+        };
+        brandInfo: {
+            name: string;
+        };
+    };
+    rank?: number;
+}
+
+const Card = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  position: relative;
+`;
+
+const RankBadge = styled.div`
+  position: absolute;
+  top: -8px;
+  left: -8px;
+  background-color: ${({ theme }) => theme.color.red.red700};
+  color: white;
+  font-weight: bold;
+  padding: 4px 8px;
+  border-radius: 999px;
+  font-size: 14px;
+`;
+
+const Image = styled.img`
+  width: 100%;
+  border-radius: 8px;
+`;
+
+const Brand = styled.span`
+  font-size: 14px;
+  color: ${({ theme }) => theme.color.semantic.textSub};
+`;
+
+const Name = styled.span`
+  font-weight: bold;
+  text-align: center;
+`;
+
+const Price = styled.p`
+  font-weight: bold;
+`;
+
+export default function ProductCard({ item, rank }: ProductCardProps) {
+    return (
+        <Card>
+            {rank !== undefined && <RankBadge>{rank}</RankBadge>}
```

Comment:
```suggestion
            {rank && <RankBadge>{rank}</RankBadge>}
```
보통 falsy한 값이 undefined를 포함하기 때문에 경우에 따라 다르겠지만 이 경우에는 이렇게 작성하셔도 무방합니다.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND 를 참고해주세요!